service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java

```diff
@@ -10,218 +10,169 @@
 import com.microsoft.azure.sdk.iot.deps.twin.TwinCollection;
 import com.microsoft.azure.sdk.iot.deps.util.Tools;
 
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 
 /**
- * The DeviceTwinDevice class represent the twin on iotHub.
- * implementing constructors and serialization functionality.
- * If object is a representation of the module twin if and only if the moduleId is set.
+ * Represent the twin on IoT hub. Implementing constructors and serialization functionality.
```

Comment:
IoT Hub (capital H) here and other places

Reply:
As I mentioned in the other comment, this is by design.